### PR TITLE
style: adjust and optimize ui and styles

### PR DESCRIPTION
**Pull request by JayJay1024**
## PR Type

What kind of change does this PR introduce?

- [x] Other... Please describe:

Adjust and optimize ui and styles.

## Why do we need this pr?

UI and styles optimization.

### CODE REVIEW

**Comment by JayJay1024** [comment]
- [ ] 
image

- [ ] 
image

File: packages/helix/pages/index.tsx [hunk 1]
```diff
@@ -41,10 +40,10 @@ function Page() {
 
   const startTime = useMemo(() => {
     const date = !dailyStatistics?.length
-      ? subMilliseconds(new Date(), secondsToMilliseconds(TIMEPAST)).getTime()
+      ? subMilliseconds(Date.now(), secondsToMilliseconds(TIMEPAST)).getTime()
       : secondsToMilliseconds(+last(dailyStatistics)!.timestamp);
 
-    return format(date, DATE_FORMAT) + ' (+UTC)';
+    return format(date, 'LLL dd, yyyy') + ' (+UTC)';
```
Comment (sxlwar) [inline packages/helix/pages/index.tsx:46]: DATE_FORMAT = LLL dd, yyyy

File: packages/helix/styles/index.scss [hunk 2]
```diff
@@ -210,9 +210,17 @@ $bg-dark: #151d35;
       padding-top: 10px;
       padding-bottom: 10px;
     }
+  }
+}
+
```
Comment (sxlwar) [inline packages/helix/styles/index.scss:215]: as above

**Review by sxlwar — CHANGES_REQUESTED** [review]

File: packages/apps/components/form-control/Direction.tsx [hunk 3]
```diff
@@ -117,59 +167,9 @@ export function Direction({
             },
           });
         }}
+        className="form-item-destination"
       />
 
-      {isBalanceLoading && (
-        <div className="absolute right-0 top-28 cursor-pointer space-x-2 text-xs">
```
[the remark this answers is not in the document]
Comment (JayJay1024) [reply packages/apps/components/form-control/Direction.tsx:123]: no need top value anymore

[the remark this answers is not in the document]
Comment (JayJay1024) [reply packages/apps/components/form-control/Direction.tsx:123]: no need top value anymore

[the remark this answers is not in the document]
Comment (sxlwar) [reply packages/apps/components/form-control/Direction.tsx:119]: 第一，结构变化，第二需要增加额外的自定义class，第三得改子组件接口。
改 top 值 只要改一处，简单点不好么？

**Review by sxlwar — CHANGES_REQUESTED** [review]

File: packages/apps/components/form-control/Direction.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (JayJay1024) [reply packages/apps/components/form-control/Direction.tsx:119]: 好了

**Review by sxlwar — APPROVED** [review]

**Pull request merged**